bee-storage/bee-storage/src/access/mod.rs

```diff
@@ -0,0 +1,25 @@
+// Copyright 2020 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+/// Holds the contract for batch access operation.
+pub mod batch;
+/// Holds the contract for delete access operation.
+pub mod delete;
+/// Holds the contract for exist access operation.
+pub mod exist;
+/// Holds the contract for fetch access operation.
+pub mod fetch;
+/// Holds the contract for insert access operation.
+pub mod insert;
+/// Holds the contract for stream access operations.
+pub mod stream;
+/// Holds the contract for truncate access operations.
+pub mod truncate;
```

Comment:
I think it would make the API simpler if we'ld make those modules private, and only use the exports below. It is unlikely that those modules will ever contain more than the corresponding trait.

Reply:
I agree, much simpler and cleaner.